### PR TITLE
Blog date appearance

### PR DESCRIPTION
**Pull request by estelle**
Make the dates italic and offset the color a bit.

### CODE REVIEW

**Review by Elchi3 — COMMENTED** [review]
This doesn't match the `.byline ` style on the posts itself. Should it?

**Comment by estelle** [comment]
the color (opacity) does. The byline, however, is not italic.

**Comment by Elchi3** [comment]
I'd say let's make both italic or neither of them.

File: assets/css/general.css [hunk 1]
```diff
@@ -114,6 +114,11 @@ li h2 {
   font-size: 2.5rem;
 }
 
+h2 + small {
+  opacity: 50%;
+  font-style: italic;
+}
```
Comment (Elchi3) [inline assets/css/general.css:120]:
```suggestion
h2 + small {
  opacity: 50%;
}
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]

**Pull request merged**